src/borg/cache.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, repository, key, manifest, path=None, sync=True, do_files=Fal
         self.key = key
         self.manifest = manifest
         self.path = path or os.path.join(get_cache_dir(), repository.id_str)
+        if os.environ.get('BORG_UNIQ_HOSTNAME'):
```

Comment:
Whatever uuid you store, it becomes not uniq once you clone a vm or something, right? I thought that was the main reason for not making this the default, otherwise we can just switch to such uuids (generatedand stored in .cache/borg on startup if not there already) and be done with it .